### ISSUE TITLE
Illegal redirect URL

### ISSUE THREAD

**Issue by mzch** [taigaio/taiga-contrib-gitlab-auth#5]
I tried to login by using this plugin, but GitLab says, "The redirect URI included is not valid.".
Yes, the redirect_uri is incorrect. It should begin `https://` but actually `https:/`
```
https://gitlab.com/oauth/authorize?client_id=5c22863d61d157a06b6f9e9c3ec0fe1590ff7f0aec9fbdd18ec5a6f3fe9c74ce&state=gitlab&response_type=code&scope=read_user&redirect_uri=https:/prjctrl.com/login?next=%252Fdiscover/login
```
How can I fix this issue?

**Comment by illusi0nary** [comment]
Your redirect URL is: https:/prjctrl.com/login?next=%252Fdiscover/login
Is that right? It should be https://prjctrl.com/login

**Comment by mzch** [comment]
No, that's not right. I don't know why redirect url is `https:/prjctl.com/login...` instead of `https://prjctl.com/...`.
Could you tell me where I should check?
